lib/extensions/brakeman_excludes_patch.rb

```diff
@@ -0,0 +1,25 @@
+module BrakemanExcludesPatch
+  # Original code can be found here:
+  # https://github.com/presidentbeef/brakeman/blob/5a1eb49216b7087243f5ed0f17c90ea8ca696df1/lib/brakeman/app_tree.rb#L204-L216
+  #
+  # Brakeman silently excludes the vendor directory as of 5.0 by default, see:
+  # https://github.com/presidentbeef/brakeman/commit/89e40fef82a2144aaec5f6203959df39aed22c17
+  # This exclusion occurs AFTER inclusion paths via configuration such as engine paths.
+  # The patch below overrides this behavior and only excludes vendor/ files that aren't opted-in via engine paths.
+  def reject_global_excludes(paths)
+    paths.reject do |path|
+      relative_path = path.relative
+
+      # Add last check for engine_paths and store the result of the if for logging
+      result = if @skip_vendor and relative_path.include? 'vendor/' and @engine_paths.none? {|p| path.absolute.include?(p)}
+        true
+      else
+        Brakeman::AppTree::EXCLUDED_PATHS.any? do |excluded|
+          relative_path.include? excluded
+        end
+      end
+      puts "Excluding: #{relative_path}" if result
```

Comment:
I don't know if we want this as it can be very noisy in the test output but previously, vendor/ files were silently being excluded.  Please review the security suite results and let me know.

Reply:
I'm thinking we should remove this line for 2 reasons.  First, it is noisy, and we likely won't actually find anything in there, because no one is going to look at it.  Second, this code will go away anyway once we get an upstream fix, and it's unlikely upstream will be printing anything, so we shouldn't try to depend on it.

Reply:
Agreed.  Done. 

I contemplated a better way to do this but it's really unfortunate this code is already looping through all files to determine what to exclude so the only thing I came up with is reporting the number excluded and that wasn't very helpful.